raiden_contracts/tests/fixtures/contracts.py

```diff
@@ -31,10 +30,10 @@ def deploy_contract_txhash(
     else:
         contract = web3.eth.contract(abi=abi, bytecode=bytecode)
 
-    mine_blocks(web3, 3)
+    web3.testing.mine(3)  # type: ignore
```

Comment:
The `mine_blocks` function got removed everywhere. As this function is nothing else but a wrapper for this line of code here (wonder why it ever was a function as it causes more lines of code due to the import, still required two parameters and doesn't provide better naming that would lead to better readability), I just replaced it in-place here. It would not be worth to keep this whole file of a single function of a single line just for using it twice only in this single file. Everywhere else we now need time traveling.